src/app/components/program-details/program-details.css

```
@@ -0,0 +1,438 @@
+.indicator-label {
+    font-size: 2rem;
+    color: var(--text-color);
+    opacity: 0.8;
+    min-height: 2.2em;
+    /* Ensure consistent height for labels */
+    display: flex;
+    /* Use flex to center text vertically if needed */
+    align-items: center;
+    justify-content: center;
+    margin-top: 0;
+    color: var(--secondary-color-light);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)